src/commands/project/create.ts

```diff
@@ -0,0 +1,46 @@
+import inquirer from 'inquirer';
+import { Flags } from '@oclif/core';
+import ProjectUtils from '../../architect/project/project.utils';
+import BaseCommand from '../../base-command';
+
+export default class ProjectCreate extends BaseCommand {
+  static description = 'Create project from existing Architect templates';
+  static aliases = ['project:create'];
+
+  static flags = {
+    ...BaseCommand.flags,
+    project: Flags.string({
+      char: 'p',
+      description: 'Architect project you would like to create',
+      required: false,
+    }),
+  };
+
+  static args = [{
+    name: 'project_name',
+    description: 'Name of your project',
+    required: true,
+  }];
+
+  async run(): Promise<void> {
+    const { flags, args } = await this.parse(ProjectCreate);
+
+    let chosen_project;
+    if (!flags.project) {
+      const answer: { chosen: any } = await inquirer.prompt([
```

Comment:
You have a function that gets the repo name. We should just pass that in as a list of choices here since its only a few lines of code now.